Data/Csv/Conversion.hs

```diff
@@ -15,6 +13,11 @@
     UndecidableInstances
     #-}
 
+#if MIN_VERSION_base(4,9,0)
```

Comment:
this doesn't match the conditional you used in `cassava.cabal`; so please sync them to both use either ghc's version or base's version as discriminator